deps/media-playback/media-playback/media.c

```diff
@@ -754,12 +966,18 @@ static void *mp_media_thread_start(void *opaque)
 			m->stop_cb(m->opaque);
 		}
 	}
-
```

Comment:
Add this blank line back. You don't need to remove pre-existing blank lines like this. Extra whitespace changes just creates extra noise in the diff.

Reply:
Done.